buildSrc/src/main/kotlin/usvm.kotlin-conventions.gradle.kts

```diff
@@ -48,7 +48,7 @@ tasks.named<Test>("test") {
     // Use JUnit Platform for unit tests.
     useJUnitPlatform()
 
-    maxHeapSize = "1G"
+    maxHeapSize = "4G"
```

Comment:
Why? Isn't it too big increase?

Reply:
2G is not enough to make tests stable. 1G is no sufficient at all (probably due to some changes in jacodb). @Saloed said that even 8G is OK for running the analyses.